apps/desktop/src/lib/components/Board.svelte

```diff
@@ -135,6 +173,28 @@
 			<BranchDropzone />
 		{/if}
 	</div>
+
+	<Modal
+		bind:this={deleteOnMergingWarningModal}
+		title="Branches are not deleted automatically"
+		type="warning"
+		width="small"
+		onSubmit={acknowledgeWarning}
+	>
+		{#snippet children()}
+			<p class="text-13 text-body helper-text">
+				It seems that branches are <strong>not automatically deleted after being merged</strong>.
+				This may lead to unexpected behavior when using stacks.
+				<br />
+				<br />
+				We <strong>strongly</strong> recommend you to configure your repository to automatically delete
+				branches after they are merged.
```

Comment:
I was thinking it could be good to portray this with a little less warning, something like this?

> Configuration required for stacked branches!
> 
> We have detected your forge is currently not configured to delete branches after they are merged, please consider doing so now.
> 
> Pull requests that depend on one another will not be automatically updated when the parent is merged if you continue without this configuration.

Reply:
Uhh, not sure I like my own suggestion. What I'm getting at is we should probably introduce the user to stacking and explain this config is needed, instead of the first experience being a stern warning?